src/main/java/org/klodnicki/repository/AccountRepository.java

```diff
@@ -31,4 +31,31 @@ public List<Account> findAllAccounts() {
 
         return query.getResultList();
     }
+
+    public void removeAccount(String pesel) {
+
+        entityManager.getTransaction().begin();
+        String hqlQuery = "DELETE FROM Account a WHERE a.pesel = :pesel";
+        Query query = entityManager.createQuery(hqlQuery);
+        query.setParameter("pesel", pesel);
+        query.executeUpdate();
+        entityManager.getTransaction().commit();
+    }
+
+    public Optional<Account> accountExist(String pesel) {
```

Comment:
Name of this method would rather suggest that it returns boolean instead of Optional<Account>. Could you change either method name or return type, please?

Reply:
OK I have changed the name of that method.